minetest/minetest
# Use "Aux" key name consistently

Fixes #7600.

This changes the inconsistent naming of the key “Aux1”/“aux”/“Aux”/“Special”/“special1”/“Use”.

In player-facing, this key is now called “Aux” everywhere.

Internal names are also changed. Internally, this is now referered to as “aux”, “AUX”, “aux1” or “AUX1”. The “1” is preserved for historical reasons only.
This also changes the setting `keymap_special` to `keymap_aux1`. So players might need to reconfigure this key after updating. It's a minor inconvenience, but not a bug.

Name “Aux” (with capital “A”) picked per core dev consensus in #7600 and elsewhere.

## To do

This PR is Ready for Review.

## How to test

Compile and run.
Look in controls.
Check out updated settings.
Read source code to spot weirdnesses.
